coreos-installer

```diff
@@ -551,58 +556,93 @@ validate_image() {
             sha256sum -c /mnt/dl/imagefile.gz.sig
             if [ $? -ne 0 ]
             then
-                dialog --title 'CoreOS Installer' --msgbox "Install Image is corrupt. Dropping to shell" 10 70
+                log "Install Image is corrupted."
                 exit 1
             fi
         else
-            dialog --title 'CoreOS Installer' --infobox "Unknown signature type $SIG_TYPE, skipping validation" 10 70
-            sleep 3
+            log "Unknown signature type ${SIG_TYPE}"
+            exit 1
         fi
     else
-        dialog --title 'CoreOS Installer' --infobox "Unknown signature type, skipping validation" 10 70
-        sleep 3
+        log "Skipping image signature validation."
     fi
-
-    dialog --clear
 }
 
 #########################################################
 #Wipe any remaining disk labels
 #########################################################
 wipe_target_disk_labels() {
-    dialog --title 'CoreOS Installer' --infobox "Wiping ${DEST_DEV}" 10 70
+    log "Wiping ${DEST_DEV}"
     wipefs --all --force "${DEST_DEV}"
 }
 
+log() {
+    echo "$1" >> /tmp/debug
+    echo "$1" >&2
+}
+
 #########################################################
 #And Write the image to disk
 #########################################################
 write_image_to_disk() {
-    echo "Writing disk image" >> /tmp/debug
+    log "Writing disk image"
     
     zcat /mnt/dl/imagefile.gz |\
-    dd bs=1M iflag=fullblock oflag=direct of="${DEST_DEV}" status=progress  |&\
-    dialog --title 'CoreOS Installer' --gauge "Writing image to disk" 10 70
-    
+    dd bs=1M iflag=fullblock oflag=direct of="${DEST_DEV}" status=none
+
     for try in 0 1 2 4; do
         sleep "$try"  # Give the device a bit more time on each attempt.
         blockdev --rereadpt "${DEST_DEV}" && unset try && break
     done
     udevadm settle
 }
 
-# Main function for running the installer
-main() {
+parse_args() {
+    USAGE="Usage: $0 [options]
+Options:
+    -d DEVICE   Install to the given device, alternatively specify coreos.inst.install_dev on the kcmdline.
+    -i IGNITION The URL (or path) to the given Ignition config, alternatively specify coreos.inst.ignition_url on the kcmdline.
+    -b BASEURL  The URL to the image, alternatively specify coreos.inst.image_url on the kcmdline.
```

Comment:
when viewing this output on a limited VGA console (i.e. virt-manager) the text flows to the next line. and looks jumbled. Maybe we should wrap at 75 characters?

Reply:
That's a good point, I'll go full pep8 and wrap at 72 chars.